File: MPI/mpicheck/tests/functions/2.0/MPI_File_get_type_extent.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_File var_0;
    MPI_Datatype var_1;
    MPI_Aint *var_2;
    int ret;
    /* calls */
    ret = MPI_File_get_type_extent(var_0, var_1, var_2);
    ret = PMPI_File_get_type_extent(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_File_get_type_extent.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER(KIND=MPI_ADDRESS_KIND) var_2
       INTEGER var_3
        call mpi_file_get_type_extent(var_0, var_1, var_2, var_3)
       call pmpi_file_get_type_extent(var_0, var_1, var_2, var_3)
        end program main
    